funnel/models/label.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this file copy-pasted from the master branch? It's showing up as a new file here.